docs/source/getting_started/public_access.rst

```diff
@@ -0,0 +1,42 @@
+=======================
+Dataset Access Control
+=======================
+
+Overview
+--------
+
+When a dataset is created on the Citrine platform, only members of the project in which it was
+created can see it and interact with it. If a dataset is made public it can be retrieved by any user
+using any project. Once retrieved, its data objects are also available to view. Datasets that are
+public may only be written to by the project from which they originated.
+
+Toggling Public Access
+----------------------
+
+A dataset can be made public (or made private again) using the project from which it originates
+using the :func:`~citrine.resources.project.Project.make_public` and :func:`~citrine.resources.project.Project.make_private` methods on the project.
+
+Example
+-------
+
```

Comment:
Add a line here to the effect "Assume you have a "band gaps project" with known uid `band_gaps_project_id` and an associated dataset with known uid `strehlow_cook_dataset_id`".

It has been my experience that unless these things are spelled out, people have looked at the docs and thought they were built-ins.

Reply:
This is excellent advice.